.github/pull_request_template.md

```diff
@@ -6,12 +6,13 @@ Please write a few sentences describing the overall goals of the pull request's
 
 ## PR checklist
 
-Please tick as appropriate and edit the reasons (e.g.: "backport: not needed because this is a new feature")
+Please add/remove strikethrough ~~ as appropriate, and add any relevant link/s to the end of the line.
 
-- [ ] **changelog** provided, or not required
-- [ ] **3.6 backport** done, or not required
-- [ ] **2.28 backport** done, or not required
-- [ ] **tests** provided, or not required
+- [ ] **changelog** provided, ~not required because:~
+- [ ] **dev PR** provided #, ~not required because:~
```

Comment:
In a pull request to `development`, it's not clear what we expect people to do. Remove the line? Leave the template alone? Edit to “this is it”? We should give some concise but clear guidance. (Not easy, I know.)

Maybe this?

```suggestion
- [ ] **development PR** here | TODO | provided # | not required because 
```

Reply:
The same as with tests. Use provided without a PR link. I will add a segment in the description if needed.